nvflare/fuel/hci/server/login.py

```diff
@@ -202,5 +202,9 @@ def pre_command(self, conn: Connection, args: List[str]):
             conn.set_prop(ConnProps.TOKEN, token)
             return True
         else:
-            conn.append_error("not authenticated - no user")
+            conn.append_error(
+                "admin client session timed out after {} seconds of inactivity - logging out".format(
+                    self.session_mgr.idle_timeout
+                )
+            )
```

Comment:
If we gets no session, the reason could be

1. The session was created, but it timed out -> then this error msg here is correct.
2. The session was never created for this client token (due to authentication) -> then this error message is misleading?

Reply:
For 2, if it fails to authenticate and thus never creates the session, wouldn't it never reach that function since it would 
'conn.append_string("REJECT")' which would throw some sort of error?

Reply:
Or we could combine the two error messages,
something like "admin client session not found, it could due to session timed out or the user is not authenticated."